<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBC</class>
 <widget class="QWidget" name="WBC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WBC</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="patchTabs_tw">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="AWB">
      <attribute name="title">
       <string>AWB</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="1" colspan="4">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Colour Correction Matrix</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0" rowspan="2" colspan="2">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QDoubleSpinBox" name="AWB_matrix0_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QDoubleSpinBox" name="AWB_matrix3_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="AWB_matrix1_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="AWB_matrix2_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_6">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="AWB_matrix4_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QDoubleSpinBox" name="AWB_offsets1_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="AWB_matrix7_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QDoubleSpinBox" name="AWB_offsets2_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QDoubleSpinBox" name="AWB_matrix6_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QDoubleSpinBox" name="AWB_matrix8_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_8">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QDoubleSpinBox" name="AWB_offsets0_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="label_7">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>x</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QDoubleSpinBox" name="AWB_matrix5_sb">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1" rowspan="2" colspan="4">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>White Balance Correction</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="AWB_gain2_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="AWB_gain0_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="AWB_gain1_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Gain</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QDoubleSpinBox" name="AWB_gain3_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Clip</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QDoubleSpinBox" name="AWB_clip0_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="AWB_clip1_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QDoubleSpinBox" name="AWB_clip2_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QDoubleSpinBox" name="AWB_clip3_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Mode (instant)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Measured Temp.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="AWB_measuredTemp_sb">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="suffix">
          <string> K</string>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="AWB_mode_lb">
         <item>
          <property name="text">
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Average Colour</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>White Patch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>High Luminance White</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Combined</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="AWB_default_btn">
         <property name="text">
          <string>Set as Default CCM</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="tune_btn">
     <property name="text">
      <string>Tune</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>patchTabs_tw</tabstop>
  <tabstop>AWB_mode_lb</tabstop>
  <tabstop>AWB_measuredTemp_sb</tabstop>
  <tabstop>AWB_default_btn</tabstop>
  <tabstop>AWB_gain0_sb</tabstop>
  <tabstop>AWB_gain1_sb</tabstop>
  <tabstop>AWB_gain2_sb</tabstop>
  <tabstop>AWB_gain3_sb</tabstop>
  <tabstop>AWB_clip0_sb</tabstop>
  <tabstop>AWB_clip1_sb</tabstop>
  <tabstop>AWB_clip2_sb</tabstop>
  <tabstop>AWB_clip3_sb</tabstop>
  <tabstop>AWB_matrix0_sb</tabstop>
  <tabstop>AWB_matrix1_sb</tabstop>
  <tabstop>AWB_matrix2_sb</tabstop>
  <tabstop>AWB_offsets0_sb</tabstop>
  <tabstop>AWB_matrix3_sb</tabstop>
  <tabstop>AWB_matrix4_sb</tabstop>
  <tabstop>AWB_matrix5_sb</tabstop>
  <tabstop>AWB_offsets1_sb</tabstop>
  <tabstop>AWB_matrix6_sb</tabstop>
  <tabstop>AWB_matrix7_sb</tabstop>
  <tabstop>AWB_matrix8_sb</tabstop>
  <tabstop>AWB_offsets2_sb</tabstop>
  <tabstop>tune_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
